tests/inverted-index-test.js

```diff
@@ -121,38 +124,119 @@ describe('Inverted index class creates index', () => {
 
     it('asserts that fileContent is valid', () => {
       expect(invertedIndex.createIndex('emptyObject.json', emptyObject))
-      .toEqual([false, 'invalid']);
+      .toEqual([false, 'Invalid!']);
       expect(invertedIndex.createIndex('emptyFile.json', emptyFile))
-      .toEqual([false, 'empty']);
+      .toEqual([false, 'Empty!']);
       expect(invertedIndex.createIndex('oneMalformedBook.json',
        oneMalformedBook))
-       .toEqual([false, 'malformed', [0]]);
+       .toEqual([false, 'Malformed!']);
     });
 
     it('creates and returns an index object', () => {
-      const index = invertedIndex.createIndex('anotherValidFile.json',
-       anotherValidFile);
-      expect(index.edge).toBeDefined();
-      expect(index.edge).toEqual([0, 1, 2]);
-      expect(index.fly).toBeDefined();
-      expect(index.fly).toEqual([1, 2]);
+      const fileIndex = invertedIndex.createIndex('anotherValidFile.json',
+       anotherValidFile)['anotherValidFile.json'];
+      expect(fileIndex.edge).toBeDefined();
+      expect(fileIndex.edge).toEqual([0, 1, 2]);
+      expect(fileIndex.fly).toBeDefined();
+      expect(fileIndex.fly).toEqual([1, 2]);
     });
 
     it('creates and returns an index object', () => {
-      const index = invertedIndex.createIndex('validFile.json',
-       validFile);
-      expect(index.animals).toBeDefined();
-      expect(index.animals).toEqual([0, 1]);
-      expect(index.system).toBeDefined();
-      expect(index.system).toEqual([1, 2]);
+      const fileIndex = invertedIndex.createIndex('validFile.json',
+       validFile)['validFile.json'];
+      expect(fileIndex.animals).toBeDefined();
+      expect(fileIndex.animals).toEqual([0, 1]);
+      expect(fileIndex.system).toBeDefined();
+      expect(fileIndex.system).toEqual([1, 2]);
+      expect(fileIndex.the).toBeDefined();
+      expect(fileIndex.the.length).toEqual(validFile.length);
     });
+  });
+});
 
-    it('creates and returns an index object', () => {
-      const index = invertedIndex.createIndex('validFile.json',
-       validFile);
-      Object.keys(index).forEach((item) => {
-        expect(index[item].includes(3)).toBeFalsy();
-      });
+describe('Inverted Index class searches an index for term(s)', () => {
+  describe('InvertedIndex.flattenArray(nestedArray)', () => {
+    it(`takes a multidimensional array of search terms 
+    and returns a one dimensional array`, () => {
+      expect(InvertedIndex
+      .flattenArray([[1, 2, 3], [[[[[4, 5], 'a', 'b']]], ['x'], 'y']]))
```

Comment:
Do you think that this nested arrays  and other related instances can be better presented on seperate lines to make it more readable?